docs/language/functions.rst

```diff
@@ -308,6 +308,39 @@ calling.
 
 .. _fallback_receive:
 
+Calling an external function using ``delegatecall``
+___________________________________________________
+
+External functions can also be called using ``delegatecall``.
+The difference to a regular ``call`` is that  ``delegatecall`` executes the callee code in the context of the caller,
+which has the following implications:
+
+* The callee will read from and write to the `caller` storage
+* ``value`` can't be specified for ``delegatecall`` (``msg.value`` stays the same in the callee)
+* ``msg.sender`` stays the same in the callee
```

Comment:
Yes but they are different things worth of pointing them out individually

Reply:
> Yes but they are different things worth of pointing them out individually

I don't understand why they are different things. Would you mind writing a few words about that in the docs?
I am a completely oblivious to Substrate, so someone who is only starting with it might wonder the same thing as I do.

Reply:
(The first sentence talks about `msg.value`, not `msg.sender`).